### DISCUSSION TITLE
SunbirdED release 5.2.0 issue with the editor and player updates

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-inQuiry/Community#63] [General Discussions]
As sunbirdEd portal is still using Angular-11 and the InQuiry releases are using Angular-12 for the editor and player here 
https://www.npmjs.com/package/@project-sunbird/sunbird-collection-editor/v/5.3.2
https://www.npmjs.com/package/@project-sunbird/sunbird-quml-player/v/5.3.0
if we take the update the compilation of portal code will not work as the versions are different so we will not be able to take the update till we migrate Portal code base to Angular-12 version

cc: @pallakartheekreddy @Krishnaj20 @vinukumar-vs @chitranshu-keshav

**Comment by ashokreddy1208** [comment]
@rajeevsathish can you please confirm if 5.3 InQuiry changes are deployed on sunbird ED are not?
cc: cc: @pallakartheekreddy @Krishnaj20 @vinukumar-vs @chitranshu-keshav @Aparna-karri591
